### ISSUE TITLE
Package assembl for sandstorm.io

### ISSUE THREAD

**Issue by jmatsushita** [ImaginationForPeople/assembl#7]
https://docs.sandstorm.io/en/latest/vagrant-spk/packaging-tutorial/

Maybe @Sing-Li would be interested? I wonder if packaging Virtuoso is possible...

**Comment by benoitg** [comment]
I'm sure it's possible, but I doubt the core team will spend time on it.  I think #3 would bring us considerably more traction.

**Comment by jmatsushita** [comment]
Indeed. Given that both use linux containers, moving forward on one will get closer to the other.
